google-youtube.html

```diff
@@ -647,6 +747,7 @@
       this.currenttime = Math.round(this._player.getCurrentTime());
       this.currenttimeformatted = this._toHHMMSS(this.currenttime);
       this.fractionloaded = this._player.getVideoLoadedFraction();
+      this.volume = this._player.getVolume();
```

Comment:
Would it be more appropriate to use `this.volume = Math.round(this._player.getVolume());` in this occasion, or should we let the user decide?